modules/gapi/include/opencv2/gapi/infer/parsers.hpp

```diff
@@ -85,11 +86,11 @@ the larger side of the rectangle.
 @param filterOutOfBounds If provided true, out-of-frame boxes are filtered.
 @return a vector of detected bounding boxes.
 */
-GAPI_EXPORTS GArray<Rect> parseSSD(const GMat& in,
-                                   const GOpaque<Size>& inSz,
-                                   const float confidenceThreshold = 0.5f,
-                                   const bool alignmentToSquare = false,
-                                   const bool filterOutOfBounds = false);
+GAPI_EXPORTS GArray<Rect> parseSSDOOB(const GMat& in,
+                                      const GOpaque<Size>& inSz,
+                                      const float confidenceThreshold = 0.5f,
+                                      const bool alignmentToSquare = false,
+                                      const bool filterOutOfBounds = false);
```

Comment:
Is there a way to keep a single method by removing some arguments? As for me, `filterOutOfBounds` is a redundant one. OOF boxes should be filtered anyway - there is no point to provide them for user.

Reply:
Needs save current interface for one of two versions.